neuralmonkey/encoders/attentive.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=too-few-public-methods
+class Attentive(object):
+    def __init__(self, attention_type, **kwargs):
+        self._attention_type = attention_type
+        self._attention_kwargs = kwargs
+
+        assert hasattr(self, "name")
+        assert hasattr(self, "_padding")
+        assert hasattr(self, "_attention_tensor")
```

Comment:
Myslel jsem to tak, ze enkoder by zavolal `super`, ale to by nic neudelalo, a teprv dal v konstruktoru, az by to bylo pripravene, by zavolal `self._init_attention()`.

Attention objekt se uz tak vytvari az volanim z dekoderu, jestli se nepletu.